charts/postgres-operator/values-crd.yaml

```diff
@@ -93,10 +93,9 @@ configKubernetes:
   # template for database user secrets generated by the operator
   secret_name_template: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
   # group ID with write-access to volumes (required to run Spilo as non-root process)
-  # spilo_fsgroup: 103
-
+  spilo_fsgroup: "103"
```

Comment:
if the fsgroup is required for the TLS feature to work, it should be spelled out.

Reply:
good idea. I already have a check in the code but I will add a note in the documentation as well.

Reply:
done!